feature/center/src/main/java/com/mifos/feature/center/sync_centers_dialog/SyncCentersDialogScreen.kt

```diff
@@ -174,14 +184,14 @@ fun SyncGroupDialogContent(
         Row(
             modifier = Modifier.fillMaxWidth()
         ) {
-            if (uiData.isSyncSuccess) {
```

Comment:
why are we deleting this logic?

Reply:
The Cancel and hide button doesnt show and hence its implementation goes for a waste. 
![Screenshot 2024-11-09 193339](https://github.com/user-attachments/assets/2c7a46be-8080-4d24-8311-f1f447d41e8b)
Do you want me to add it back @itsPronay

Reply:
`hide` and `cancel` should show when uiData.isSyncSuccess is false. If its not showing when syncing, the problem must be somewhere else.

Reply:
keep it as it was